synapse/util/metrics.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, clock, name):
             assert isinstance(curr_context, LoggingContext)
             parent_context = curr_context
         self._logging_context = LoggingContext(
-            "Measure[%s]" % (self.name,), parent_context
+            self.name, parent_context
```

Comment:
this looks wrong - the `request` that gets logged inside a `Measure` block should be the same as the one that gets logged outside it - so the `name` should be `parent_context.name`

Reply:
So the `name` here is the *metric* name, not the logging context name? 😄

Reply:
`self.name` is the metric name, yes.